Java.gitignore

```diff
@@ -1,5 +1,13 @@
 *.class
 
+# Eclipse editor
```

Comment:
We have an existing [`Global\Eclipse.gitignore`](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore). Please drop these rules.